digits/inference/errors.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from digits.utils import subclass
+
+@subclass
```

Comment:
FYI `@subclass` doesn't really do anything if you're not also using `@override`. But I'm fine with leaving it in if you think it makes the code more readable.